src/linagora.esn.controlcenter/app/general/controlcenter-general.controller.js

```diff
@@ -1,3 +1,5 @@
+const _ = require('lodash');
```

Comment:
You just added the _ provided by webpack. You should remove the _ provided through AngularJS injection (
```
function controlcenterGeneralController(
    $q,
    $window,
    esnUserConfigurationService,
    asyncAction,
    homePageService,
    _, // <- REMOVE HERE
    ESN_ROUTER_DEFAULT_HOME_PAGE,
    CONTROLCENTER_GENERAL_CONFIGS
  ) {
```